.travis.yml

```diff
@@ -24,6 +24,9 @@ env:
   - TESTS="1" ODOO_REPO="odoo/odoo" MAKEPOT="1"
   - TESTS="1" ODOO_REPO="OCA/OCB"
 
+virtualenv:
```

Comment:
I doubt this is the way to resolve this. We try to prevent the need for system site packages wherever possible to create a really isolated environment. @hbrunn you have an opinion on this one?

Reply:
I'm the author of this. v10 uses a legacy version of python, so I also don't nee a problem to use a legacy python option here